src/drt/src/distributed/RoutingCallBack.h

```diff
@@ -53,37 +55,60 @@ class RoutingCallBack : public dst::JobCallBack
   }
   void onRoutingJobReceived(dst::JobMessage& msg, dst::socket& sock) override
   {
-    if (msg.getType() != dst::JobMessage::ROUTING)
+    if (msg.getJobType() != dst::JobMessage::ROUTING)
       return;
+    dst::JobMessage result(dst::JobMessage::ROUTING);
     RoutingJobDescription* desc
         = static_cast<RoutingJobDescription*>(msg.getJobDescription());
-    if (access(desc->getWorkerPath().c_str(), F_OK) == -1) {
-      logger_->warn(
-          utl::DRT, 605, "Worker file {} not found", desc->getWorkerPath());
+    if (desc->getWorkerPath() != "") {
+      if (access(desc->getWorkerPath().c_str(), F_OK) == -1) {
+        logger_->warn(
+            utl::DRT, 605, "Worker file {} not found", desc->getWorkerPath());
```

Comment:
How would the leader know of this error?

Reply:
It wouldn't for now because there is no error handling in the current version, but we should support error results sent back to the leader in the future.